doc/OnlineDocs/source/tasks/install_solvers.rst

```diff
@@ -0,0 +1,63 @@
+Installing Specific Linear Solvers
+==================================
+
+Prescient uses a mixed-integer linear programming (MILP) solver to make
+dispatch and unit commitment decisions. Before running Prescient, a solver
+must be installed and available to `Pyomo <https://pyomo.readthedocs.io>`_.
+Installation guidance for a number of common solvers is found below.
+
+.. contents:: Solvers
+   :local:
+
+The choice of solver often comes down to cost. CBC is free, but is typically
+slower than commercial alternatives. Commercial solvers are faster, but
+require a license and may require additional configuration. Some solvers
+offer free licenses for academic or research use. Check with the appropriate
+vendor for details.
+
+CBC
+---
+CBC is a free, open-source linear solver. On Linux and Mac platforms, CBC
+can be installed using Anaconda::
+
+	conda install -c conda-forge coincbc
+
+Binaries for additional platforms, including Windows, may be available from
+https://github.com/coin-or/Cbc/releases. When installing CBC manually, you
+may need to modify your path to ensure the cbc executable is available on
+the command line.
+
+CBC is the solver Prescient uses for automated tests on github.
+
+Gurobi
+------
+
+Python bindings for Gurobi can be installed via conda::
+
+	conda install -c gurobi gurobi
+
+Gurobi requires a valid license. See the documentation on the 
+`Gurobi <https://gurobi.com>`_ website for instructions on how to
+acquire and install a license.
+
+Depending on your license, you may not be able to use the latest version
+of the solver. A specific release can be installed as follows::
+
+	conda install -c gurobi gurobi=8
```

Comment:
We should probably direct users to PyPI like Xpress below: https://pypi.org/project/gurobipy/

Reply:
I added conda and pip (PyPI) instructions for solvers available from both sources.